client/app/states/blueprints/designer/designer.html

```diff
@@ -96,7 +96,7 @@
                        data-jqyoui-options="{revert: 'invalid', helper: 'clone'}"
                        ng-click="addNodeByClick( getNewItem() )">
                       <span class="pficon pficon-add-circle-o"></span>
-                      <span class="new-catalog-item-label">{{'Create ' + getNewItem().title | translate}}</span>
+                      <span class="new-catalog-item-label">{{'Create ' + getNewItem().name | translate}}</span>
```

Comment:
This needs to be changed to

```html
<span class="new-catalog-item-label">{{'Create'| translate}} {{getNewItem().name}}</span>
```

Otherwise the translation here won't work.